config.xml

```diff
@@ -262,4 +263,7 @@
   <doc cat="styleguide" doc="docu_styleguide" branches="main">Style Guide</doc>
   <doc cat="styleguide" doc="docu_styleguide_with_changes" branches="main">Style Guide with Change Log</doc>
 
+  <doc cat="release-notes" doc="sles" branches="15_SP4 15_SP3">SUSE Linux Enterprise Server</doc>
+  <doc cat="release-notes" doc="sled" branches="15_SP4 15_SP3">SUSE Linux Enterprise Desktop</doc>
```

Comment:
:/ I had a thinko yesterday when I asked you to change the URL.

This is a random existing config + resulting URLs:
```
<doc cat="sles"  doc="SLES-administration" branches="main SLE15SP3">Administration Guide</doc>
-> https://susedoc.github.io/doc-sle/main/html/SLES-administration/
-> https://susedoc.github.io/doc-sle/main/single-html/SLES-administration/
```

1. There are two formats (`/html/`, `/single-html/`), that will have to be fixed in this repo, because the release notes only exist as single-HTMLs. (Whether you drop them under `/html/` or `/single-html/` is of no importance to me.
2. For the script, there currently needs to be an extra component at the end still. In the example above, that is `SLES-administration`. I am a bit more reluctant to special-case that.

So, if we assume you are happy to add another subdir `/release-notes/` under the existing `/html/` dir, we could go for:

```diff
-  <doc cat="release-notes" doc="sles" branches="15_SP4 15_SP3">SUSE Linux Enterprise Server</doc>
-  <doc cat="release-notes" doc="sled" branches="15_SP4 15_SP3">SUSE Linux Enterprise Desktop</doc>
+  <doc cat="release-notes" doc="release-notes" branches="sles-15_SP4 sles-15_SP3">SUSE Linux Enterprise Server</doc>
+  <doc cat="release-notes" doc="release-notes" branches="sled-15_SP4 sled-15_SP3">SUSE Linux Enterprise Desktop</doc>
```

The URL would then be `https://susedoc.github.io/release-notes/sled-15_SP4/html/release-notes/`.

On a scale of 1-10, how unhappy are you about adding another useless dir structure?

Reply:
I think I am OK with it for now :) It looks terrible but let's make it work first.

Reply:
One thing I'll note is confusing is the branch naming in the `branches=` attribute but I suppose there's no getting around that?

Reply:
I was trying to avoid having to do updates to the XSLT script. Obviously, all of that is fixable but I don't think I have time for it right now.

One more thing: you can define redirects, e.g. susedoc.github.io/r/sles-154 that can redirect to the proper page if you want a short shareable URL.

Reply:
I understand :) I will update the PR to include your suggestion above and let's get that in first.